{{cookiecutter.project_slug}}/ansible/deploy.yml

```diff
@@ -7,49 +7,83 @@
 - name: deploy new version of the project to the server
   hosts: web
   remote_user: buildout
+
   tasks:
 
-    - name: checkout correct version from github
+    - name: Checkout correct version from github.
       git:
         accept_hostkey: yes
-        dest: /srv/{% raw %}{{ site_name }}{% endraw %}
-        repo: ssh://git@github.com/nens/{{ cookiecutter.project_slug }}.git
-        version: "{% raw %}{{ checkout_name }}{% endraw %}"
+        dest: {{ project_path }}
+        repo: ssh://git@github.com/nens/{{ project_name }}.git
+        version: "{{ checkout_name }}"
 
-    - name: symlink buildout config
+    - name: Create var dirs if not available.
       file:
-        src: /srv/{% raw %}{{ site_name }}/{{ buildout_cfg }}{% endraw %}
-        dest: /srv/{% raw %}{{ site_name }}{% endraw %}/buildout.cfg
-        state: link
+        path: {{ project_path }}/{{ item }}
+        owner: buildout
+        group: buildout
+        mode: 0755
+        state: directory
+      with_items:
+        - etc
+        - var
+        - var/static
+        - var/log
+        - var/media
 
-    - name: bootstrap buildout (when needed)
-      shell: python3 bootstrap.py
+    - name: "Deploy (look at var/log/pipenv.log if something goes fishy)"
+      shell: pipenv install --deploy --verbose > var/log/pipenv.log 2>&1
+      environment:
+        PIPENV_VENV_IN_PROJECT: 1  # pipenv install --local is coming and will replace this
```

Comment:
Ah! `--local`. Sounds good.

(Things like this is why I've got this cookiecutter project in my github notifications list! :-) )